packages/igx-templates/igx-ts/category-chart/default/index.ts

```diff
@@ -14,7 +14,7 @@ class IgxCategoryChartTemplate extends IgniteUIForAngularTemplate {
 			{ import: "IgxCategoryChartModule", from: "igniteui-angular-charts" },
 			{ import: "FormsModule", from: "@angular/forms" }
 		];
-		this.packages = ["tslib@^1.7.1", "igniteui-angular-core@~9.1.2", "igniteui-angular-charts@~9.1.2"];
+		this.packages = ["igniteui-angular-core@~9.1.2", "igniteui-angular-charts@~9.0.1"];
```

Comment:
After the merge from master to vNext, the packages are 9.1.2. You can revert that fix. Why tslib is removed?

Reply:
**tslib** package is a part of of the project's **package.json** files back from the Angular 7 update, thus there is no need to be duplicated in this.packages array.
 
For reference: https://github.com/IgniteUI/igniteui-cli/pull/415/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.ts&file-filters%5B%5D=No+extension